simple-local-avatars.php

```diff
@@ -472,6 +474,48 @@ public function unique_filename_callback( $dir, $name, $ext ) {
 	public function user_profile_update_errors( WP_Error $errors ) {
 		$errors->add( 'avatar_error', $this->avatar_upload_error );
 	}
+
+	/**
+	 * Registers the simple_local_avatar field in the REST API.
+	 */
+	public function register_rest_fields() {
+		register_rest_field( 'user', 'simple_local_avatar', array(
+			'get_callback' => array( $this, 'get_avatar_rest' ),
+			'update_callback' => array( $this, 'set_avatar_rest' ),
+			'schema' => array(
+				'description' => 'The users simple local avatar',
+				'type' => 'object',
+			)
+		));
+	}
+	
+	/**
+	 * Returns the simple_local_avatar meta key for the given user.
+	 * 
+	 * @param object $user User object
+	 */
+	public function get_avatar_rest( $user ) {
+		$local_avatar = get_user_meta( $user['id'], 'simple_local_avatar', true );

```

Comment:
So this does seem to be working, but I would have thought it would need to be `$user->ID` since it's (allegedly) an object?

Reply:
Ah I see, so it is actually an array during this process. The docblock should reflect that :)